libs/pages/settings/src/lib/feature/page-organization-ai-copilot-feature/page-organization-ai-copilot-feature.spec.tsx

```diff
@@ -0,0 +1,9 @@
+import { render } from '__tests__/utils/setup-jest'
```

Comment:
This one is deprecated 
```suggestion
import { renderWithProviders } from '@qovery/shared/util-tests'
```